pgmpy/base/DAG.py

```diff
@@ -85,6 +89,38 @@ def __init__(self, ebunch=None, latents=set()):
             out_str += "".join([f"({u},{v}) " for (u, v) in cycles])
             raise ValueError(out_str)
 
+            ebunch, latents, err_corr, err_var = parse_lavaan(kwargs["lavaan_str"])
+
+            # Call the parent __init__ with the arguments
+            super(DAG, self).__init__(ebunch=ebunch, latents=latents, err_corr=err_corr)
```

Comment:
I don't think nx.DiGraph accepts `err_corr`. Does it just ignore it?

Reply:
Fixed in 65a808846a5740fe2004b51b83c334e39c04484b